scripts/subdriver/gen-snmp-subdriver.sh

```diff
@@ -270,14 +272,8 @@ generate_C() {
 	done < ${STRWALKFILE} >> ${CFILE}
 
 	# append footer
-	cat >> "$CFILE" <<-EOF
-
-		/* end of structure. */
-		{ NULL, 0, 0, NULL, NULL, 0, NULL }
-	};
-
-	mib2nut_info_t	${LDRIVER} = { "${LDRIVER}", ${UDRIVER}_MIB_VERSION, NULL, NULL, ${LDRIVER}_mib, ${UDRIVER}_DEVICE_SYSOID };
-	EOF
+	printf "\n\t/* end of structure. */\n\t{ NULL, 0, 0, NULL, NULL, 0, NULL }\n};\n\n" >> "$CFILE"
+	printf "mib2nut_info_t	${LDRIVER} = { \"${LDRIVER}\", ${UDRIVER}_MIB_VERSION, NULL, NULL, ${LDRIVER}_mib, ${UDRIVER}_SYSOID };\n" >> "$CFILE"
```

Comment:
Out of curiosity: why a cumbersome `printf` and lots of file appendings instead of the single-operation append that was here before? To ensure tab chars - I think the `cat` can handle that too (especially with verbatim mode, `cat << 'EOF'` style)?

Reply:
Indeed, formatting purpose...